action.yml

```diff
@@ -27,17 +27,48 @@ inputs:
     required: false
     default: 'run'
   show-disk-usage-on-error:
-    descriptions: Whether to return the used disk space on failing.
+    description: Whether to return the used disk space on failing.
     required: false
     default: false
+  snakemake-version:
+   description: Which snakemake version to use
+   required: false
+   default: 8.25.5
```

Comment:
So, if we fix it like this, it will always stick to this instead of using the latest? Isn't it more desirable to use the latest version per default?

Reply:
Yes, good point.